DTI.py

```diff
@@ -153,25 +159,40 @@ def main():
 
                 print('Basic Information')
                 print('-----------------')
-                print(f'    {Fore.RESET}Username               {Fore.GREEN}{user_name}')
-                print(f'    {Fore.RESET}User ID                {Fore.GREEN}{user_id}')
-                print(f'    {Fore.RESET}Creation Date          {Fore.GREEN}{creation_date}')
-                print(f'    {Fore.RESET}Avatar URL             {Fore.GREEN}{avatar_url if avatar_id else ""}')
-                print(f'    {Fore.RESET}Token                  {Fore.GREEN}{token}')
+                print(
+                    f'    {Fore.RESET}Username               {Fore.GREEN}{user_name}')
+                print(
+                    f'    {Fore.RESET}User ID                {Fore.GREEN}{user_id}')
+                print(
+                    f'    {Fore.RESET}Creation Date          {Fore.GREEN}{creation_date}')
+                print(
+                    f'    {Fore.RESET}Avatar URL             {Fore.GREEN}{avatar_url if avatar_id else ""}')
+                print(
+                    f'    {Fore.RESET}Token                  {Fore.GREEN}{token}')
```

Comment:
Maybe you're taking the formatting TOO seriously?
The old format is even easier to read

Reply:
Reverted in commit bf3b2c402fbcfbf596999b1e01466e9eb09e8fba